images/macos/toolsets/toolset-11.0.json

```diff
@@ -11,16 +11,23 @@
             "6.12.0.93"
         ],
         "ios-versions": [
-            "14.2.0.12", "14.0.0.0", "13.20.2.2"
+            "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2"
         ],
         "mac-versions": [
-            "6.20.2.2"
+            "6.22.1.26", "6.20.2.2"
         ],
         "android-versions": [
-            "11.0.2.0"
+            "11.1.0.17", "11.0.2.0"
         ],
-        "bundle-default": "6_12_0",
+        "bundle-default": "6_12_3",
```

Comment:
Same here:
We mustn't update bundle-default because the latest bundle is not compatible with default Xcode

Reply:
Thank you, done